West Midlands | ITP-MAY-25 | Abdullah Saleh | Wireframe | Week2

3bdullah-saleh opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [✓] I have committed my files one by one, on purpose, and for a reason
- [✓] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [✓] I have tested my changes
- [✓] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [✓] My changes meet the [requirements](./README.md) of this task

## Changelist

This PR has changes in the HTML and CSS files. I made many commits in this PR to track the changes.

## Questions

Ask any questions you have for your reviewer.

illicitonion reviewed (commented):
This is generally looking good, the only issue with it is the footer isn't fixed in place - you need to scroll to see it.

Also, have a read about [task lists](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists) - your ticks in your PR description are cute, but just using an `x` will work and is more conventional :)

3bdullah-saleh commented:
I've updated the footer so it's now fixed in place — should be working as expected without needing to scroll.
Also noted on the task list formatting — I'll switch to using x for a more conventional approach in future PRs. Appreciate the heads-up!

illicitonion reviewed (commented):
The fixed looks good, but doesn't quite meet the wireframe - the footer is meant to be centred :)

Also, sometimes depending on where I'm scrolled it's hard to read the footer because of overlap with the text of the page:

<img width="420" alt="Screenshot 2025-05-20 at 10 28 26" src="https://github.com/user-attachments/assets/d5645189-c806-4d04-a069-b58163873db8" />

Can you work out how to make sure you can always read the footer (possibly inspired by the wireframe)?

3bdullah-saleh commented:
I've made the following updates based on your suggestions:

The footer is now properly centered.

Added a black top border to better match the wireframe design.

Added bottom padding to the main content to prevent overlap and improve footer readability on scroll.

illicitonion commented:
Perfect - thanks!
